src/evaluateXPath.ts

```diff
@@ -16,9 +16,10 @@ import { Node } from './types/Types';
 export type Options = {
 	currentContext?: any;
 	debug?: boolean;
-	documentWriter?: IDocumentWriter;
 	disableCache?: boolean;
+	documentWriter?: IDocumentWriter;
 	language?: Language;
+	logOutput?: (message: string) => void;
```

Comment:
Have we really considered the name of this option?
All other logging e.g. errors when using `debug: true` will still be logged in the console and raised as error.

I propose to either rename this to `traceOutput` or also use it for errors for which we'd also need an indicator for the error.

Reply:
The options has been changed to an object `logger` which can contain multiple functions to override the standard logger. As for now, you can only override the logger for the `trace()` function.